Projector.cs

```diff
@@ -0,0 +1,25 @@
+﻿// This is the Projector Abstract class
+// Abstract means the class can only be accessed through a child class. 
+// Abstract methods, can only be used by an abstract class, doesn't have a body, only provided from inheritence 
+// This class will contain all common RS 232 commands across all projectors
+
+// The abstract class's methods can only be accesed through inheritence 
+
+using System;
+
+namespace RS232_API
+{
+    abstract public class Projector
+    {
+        protected String Make;
+        protected String Model; 
+        public abstract void ON();
+        public abstract void OFF();
+        public abstract void Generation();
+        public abstract void Name();
+
+        

```

Comment:
Part of cleaning up the code: remove all these empty lines

Reply:
I edited the code such that there are no empty lines or comments.